08-19 21:35:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Index "CLASSNAME_INDEX" not found; SQL statement:
DROP INDEX `className_index` [42112-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.DropIndex.update(DropIndex.java:49)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:198)
	at com.j256.ormlite.jdbc.JdbcCompiledStatement.runExecute(JdbcCompiledStatement.java:62)
	at com.j256.ormlite.table.TableUtils.doStatements(TableUtils.java:462)
	at com.j256.ormlite.table.TableUtils.doDropTable(TableUtils.java:273)
	at com.j256.ormlite.table.TableUtils.dropTable(TableUtils.java:151)
	at mage.cards.repository.CardRepository.<init>(CardRepository.java:84)
	at mage.cards.repository.CardRepository.<clinit>(CardRepository.java:59)
	at mage.remote.SessionImpl.updateDatabase(SessionImpl.java:433)
	at mage.remote.SessionImpl.access$600(SessionImpl.java:92)
	at mage.remote.SessionImpl$4.run(SessionImpl.java:256)
	at mage.remote.SessionImpl.handleRemotingTaskExceptions(SessionImpl.java:136)
	at mage.remote.SessionImpl.connect(SessionImpl.java:239)
	at Main$.delayedEndpoint$Main$1(Main.scala:60)
	at Main$delayedInit$body.apply(Main.scala:42)
	at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
	at scala.App$class.main(App.scala:76)
	at Main$.main(Main.scala:42)
	at Main.main(Main.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at sbt.Run.invokeMain(Run.scala:67)
	at sbt.Run.run0(Run.scala:61)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:51)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Logger$$anon$4.apply(Logger.scala:84)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
08-19 21:35:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Index "NAME_INDEX" not found; SQL statement:
DROP INDEX `name_index` [42112-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.DropIndex.update(DropIndex.java:49)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:198)
	at com.j256.ormlite.jdbc.JdbcCompiledStatement.runExecute(JdbcCompiledStatement.java:62)
	at com.j256.ormlite.table.TableUtils.doStatements(TableUtils.java:462)
	at com.j256.ormlite.table.TableUtils.doDropTable(TableUtils.java:273)
	at com.j256.ormlite.table.TableUtils.dropTable(TableUtils.java:151)
	at mage.cards.repository.CardRepository.<init>(CardRepository.java:84)
	at mage.cards.repository.CardRepository.<clinit>(CardRepository.java:59)
	at mage.remote.SessionImpl.updateDatabase(SessionImpl.java:433)
	at mage.remote.SessionImpl.access$600(SessionImpl.java:92)
	at mage.remote.SessionImpl$4.run(SessionImpl.java:256)
	at mage.remote.SessionImpl.handleRemotingTaskExceptions(SessionImpl.java:136)
	at mage.remote.SessionImpl.connect(SessionImpl.java:239)
	at Main$.delayedEndpoint$Main$1(Main.scala:60)
	at Main$delayedInit$body.apply(Main.scala:42)
	at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
	at scala.App$class.main(App.scala:76)
	at Main$.main(Main.scala:42)
	at Main.main(Main.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at sbt.Run.invokeMain(Run.scala:67)
	at sbt.Run.run0(Run.scala:61)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:51)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Logger$$anon$4.apply(Logger.scala:84)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
08-19 21:35:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Index "SETCODE_CARDNUMBER_INDEX" not found; SQL statement:
DROP INDEX `setCode_cardNumber_index` [42112-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.DropIndex.update(DropIndex.java:49)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:198)
	at com.j256.ormlite.jdbc.JdbcCompiledStatement.runExecute(JdbcCompiledStatement.java:62)
	at com.j256.ormlite.table.TableUtils.doStatements(TableUtils.java:462)
	at com.j256.ormlite.table.TableUtils.doDropTable(TableUtils.java:273)
	at com.j256.ormlite.table.TableUtils.dropTable(TableUtils.java:151)
	at mage.cards.repository.CardRepository.<init>(CardRepository.java:84)
	at mage.cards.repository.CardRepository.<clinit>(CardRepository.java:59)
	at mage.remote.SessionImpl.updateDatabase(SessionImpl.java:433)
	at mage.remote.SessionImpl.access$600(SessionImpl.java:92)
	at mage.remote.SessionImpl$4.run(SessionImpl.java:256)
	at mage.remote.SessionImpl.handleRemotingTaskExceptions(SessionImpl.java:136)
	at mage.remote.SessionImpl.connect(SessionImpl.java:239)
	at Main$.delayedEndpoint$Main$1(Main.scala:60)
	at Main$delayedInit$body.apply(Main.scala:42)
	at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
	at scala.App$class.main(App.scala:76)
	at Main$.main(Main.scala:42)
	at Main.main(Main.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at sbt.Run.invokeMain(Run.scala:67)
	at sbt.Run.run0(Run.scala:61)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:51)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Logger$$anon$4.apply(Logger.scala:84)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
08-19 21:35:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CARD" not found; SQL statement:
DROP TABLE `card`  [42102-187]
08-19 21:36:01 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "EXPANSION" not found; SQL statement:
DROP TABLE `expansion`  [42102-187]
